### PR TITLE
docs: terminate lifelines in sequence diagrams

### PR DESCRIPTION
**Pull request by Sukkaito**

### CODE REVIEW

File: docs/img/SequenceDiagrams/LoadRecipesSequenceDiagram.png
Comment (seanngja) [inline docs/img/SequenceDiagrams/LoadRecipesSequenceDiagram.png:1]: What happened to the lifeline of Scanner lol

Comment (Sukkaito) [reply docs/img/SequenceDiagrams/LoadRecipesSequenceDiagram.png:1]: If you zoom in, you can see its lifeline. I was confusing at first too! 🤣

Comment (seanngja) [reply docs/img/SequenceDiagrams/LoadRecipesSequenceDiagram.png:1]: Ah I see haha, then its all good.

**Review by seanngja — APPROVED** [review]
LGTM!

**Pull request merged**